components/developer/jenkins/files/Makefile

```diff
@@ -0,0 +1,11 @@
+JENKINS_VERSION ?= 2.65
+JENKINS_RELEASE ?= weekly
+all: build
+
+build: 
+
+install: 
+	mkdir -p $(DESTDIR)/usr/share/jenkins/war
+	mkdir -p $(DESTDIR)/usr/share/jenkins/war-$(JENKINS_RELEASE)
```

Comment:
We should ship this under /usr/share/jenkins/war/weekly or /usr/share/jenkins/weekly. Does this setup mimics some practices in other distro?